app/src/main/kotlin/com/antosik/benchproject/app/common/binding/imageView/ImageViewBindingAdapters.kt

```diff
@@ -10,3 +10,8 @@ fun ImageView.loadMovieImage(imageUrl: String?) {
         .load(imageUrl)
         .into(this)
 }
+
+@BindingAdapter("app:tint")
+fun ImageView.setImageTint(color: Int) {
```

Comment:
You could add `@ColorRes` annotation before `color: Int` 😄

Reply:
Done :D